internal/attestation/crafter/materials/testdata/csaf_security_advisory.json

```diff
@@ -0,0 +1,319 @@
+{
+  "document": {
+    "aggregate_severity": {
+      "namespace": "https://access.redhat.com/security/updates/classification/",
+      "text": "Critical"
+    },
+    "category": "csaf_security_advisory",
```

Comment:
shouldn't you check for the `category`?

Reply:
The problem with the category is that it would be just a best effort since it's not a compulsory parameter.

Reply:
but as you said in another comment, should we check that it's either the value we want or empty?

Reply:
We could do that but it won't guarantee us that the document is valid and that would need to be done on the `Craft` method rather than on the initializer.

Reply:
Oki, changes applied I believe it shall be ok now.